source/fundamentals/typescript.txt

```diff
@@ -79,81 +80,51 @@ You can find a code snippet that shows how to specify a type for the ``FindCurso
 class in the
 :ref:`Find Multiple Documents Usage Example <node-driver-find-usage-example-code-snippet>`.
 
+.. _node-driver-limitations:
+
 Limitations
 -----------
 
-.. _node-driver-typescript-limitations-dot-notation:
+.. _node-driver-recursive-type:
 
-The driver cannot infer the type of values with keys containing **dot
-notation**. Dot notation is a property access syntax for navigating BSON objects.
-Click on the tabs to see code snippets that highlight this behavior:
+You cannot specify a recursive type as a type parameter for {+driver-long+}
+classes. Click the following tabs to see a recursive type and a code snippet that
+uses the recursive type and raises an error: 
```

Comment:
There are too many conjunctions in this sentence. Could you split it into two? On a related note:

I think we should refrain from using tabs (and possibly expecting readers to identify them as such) to show two or more disparate items (in this case, a type declaration and code snippet that implements the declaration). Throughout our docs, we generally only use them to  show how to accomplish one task using different tools or drivers because only one of them applies to their use case.
Not only are they different items, they cannot view both at the same time (without opening another browser) which might be easier to read. Keep in mind that some readers print out the documentation, so it's a disadvantage not to be able to see some of the information. I see some examples of this disparity in MongoDB Analyzer docs as well (e.g. "Code Snippet" and "Warning" which you might consider revisiting.

Reply:
I removed the tabs and updated the sentence. I am open to updating other content that includes tabs, but I think it might be good to have a concrete guideline accepted across docs teams to make the update against before updates are made.

I do not believe the style guide takes a stance on the usage of tabs. The only content I could find [dealing with tabs was a tutorial describing their usage](https://docs.mongodb.com/meta/tutorials/tabbed-content/). Would it make sense to add a "Tabs" entry to the style guide describing proper usage, and then file tickets according to the guidelines arrived at?